app/src/docs/_examples/pickerInput/PickerWithChangingItemsArray.example.tsx

```diff
@@ -1,17 +1,27 @@
-import React, { useState } from "react";
+import React, { useMemo, useState } from "react";
 import { demoData } from "@epam/uui-docs";
-import { useArrayDataSource } from "@epam/uui-core";
+import { ArrayDataSource } from '@epam/uui-core';
 import {PickerInput, MultiSwitch, FlexRow, FlexCell} from "@epam/promo";
 
 const fullLevelsList = demoData.languageLevels;
 const shortLevelsList = demoData.languageLevels.slice(5);
+const multiSwitchItems = [
+    {
+        id: fullLevelsList,
+        caption: 'Full levels list',
+    },
+    {
+        id: shortLevelsList,
+        caption: 'Short levels list',
+    },
+];
 
 export default function LanguageLevelsArraySinglePicker() {
-    const [pickerValue, setPickerValue] = useState<string>();
-    const [items, setItems] = useState(fullLevelsList);
+    const [pickerValue, setPickerValue] = useState<string>(null);
+    const [items, setItems] = useState([]);
```

Comment:
Why we remove default value? User should work with this example without required selection in multi switch.